features/base/participants/actions.js

```diff
@@ -9,6 +9,45 @@ import {
 } from './actionTypes';
 import './reducer';
 
+
+/**
+ * Returns the URL of the image for the avatar of a particular user,
+ * identified by its id.
+ * @param {string} userId user id
+ * @param {string{ email user email
+ */
+function getAvatarUrl(userId, email) {
+    // TODO: Use disableThirdPartyRequests config
+
+    let avatarId = email || userId;
+
+    // If the ID looks like an email, we'll use gravatar.
+    // Otherwise, it's a random avatar, and we'll use the configured
+    // URL.
+    let random = !avatarId || avatarId.indexOf('@') < 0;
+
+    if (!avatarId) {
+        avatarId = userId;
+    }
+    avatarId = MD5.hexdigest(avatarId.trim().toLowerCase());
```

Comment:
I don't see any imports for MD5. Is it in global scope? If yes, then we need to add comment with /** global */